cloud/providers/ec2/ec2_util.go

```diff
@@ -282,25 +282,44 @@ func attachTags(ec2Handle *ec2.EC2,
 	return err
 }
 
-// determine how long until a payment is due for the specified host. since ec2
-// bills per full hour the host has been up this number is just how long until,
-// the host has been up the next round number of hours
-func timeTilNextEC2Payment(host *host.Host) time.Duration {
+func timeTilNextEC2Payment(h *host.Host) time.Duration {
+	if usesHourlyBilling(h) {
+		return timeTilNextHourlyPayment(h)
+	}
+	return 1 * time.Second
```

Comment:
multiplying by one is weird. 

Also, I think that this should use some fraction of the startup time for a host? having some excess capacity for a bit means that we have low latency, and there's some base cost for a starting up a host, so we should let hosts stick around for maybe 15 minutes or maybe shorter if pools are big, and longer if the pools are smaller?

Reply:
We discussed this - the current implementation of the monitor will terminate hosts after 15 minutes of idle time, which feels about right